services/navigation/builtin/builtin_test.go

```diff
@@ -589,10 +592,6 @@ func TestNavSetup(t *testing.T) {
 	err = ns.RemoveWaypoint(ctx, id, nil)
 	test.That(t, err, test.ShouldBeNil)
 
-	obs, err := ns.Obstacles(ctx, nil)
```

Comment:
why is this test case removed?

Reply:
this test sets up a new navigation service from the `nav_cfg.json` config.
In the config, we define a single obstacle detector.
If we were to call `ns.Obstacles` we get the following error.
```
vision model "rdk:service:vision/blue_square" does not implement a 3D segmenter
```
I do not think the solution is to change the vision service to something which implements a 3D segmenter since we would still encounter an issue with not being able to get a reading from the camera. 
Hence, this case has been removed.